src/Operators/finitedifference.jl

```diff
@@ -1320,6 +1320,204 @@ Base.@propagate_inbounds function stencil_right_boundary(
     stencil_interior(op, loc, space, idx - 1, hidx, velocity, arg)
 end
 
+struct LinVanLeerC2F{BCS} <: AdvectionOperator
+    bcs::BCS
+end
+
+"""
+    LimiterConstraint
+Following the van Leer class of limiters as noted in 
+[Lin1994](@cite), four limiter constraint options are provided
+- AlgebraicMean: Algebraic mean, this guarantees neither positivity nor monotonicity (eq 2)
+- PosDef: Positive-definite with implicit diffusion based on local stencil extrema (eq 3b, 3c, 5a, 5b)
+- Mono4: Monotonicity preserving harmonic mean, this implies a strong monotonicity constraint (eq 4)
+- Mono5: Monotonicity preserving, with extrema bounded by the edge cells in the stencil (eq 5)
+
+The diffusion implied by these methods is proportional to the local upwind CFL number.
+The `mismatch` Δ𝜙 = 0 returns the first-order upwind method. Special cases (discussed in Lin et al (1994)) include setting the 𝜙_min = 0 or 𝜙_max = saturation mixing ratio for water vapor
+are not considered here in favour of the generalized local extrema in equation (5a, 5b).
+"""
+abstract type LimiterConstraint end
+struct AlgebraicMean <: LimiterConstraint end
+struct PosDef <: LimiterConstraint end
+struct Mono4 <: LimiterConstraint end
+struct Mono5 <: LimiterConstraint end
```

Comment:
`Mono4` and `Mono5` aren't very clear names. Maybe something like `HarmonicMean` and `LocallyMonotonic` would be better? Also you can probably write out `PositiveDefinite` instead of `PosDef`; it's okay to be more verbose in these names.

Reply:
These names currently match the terminology in the reference, but I agree verbose names would be helpful - with a docstring reference to the paper terminology.